CONTRIBUTING.md

```diff
@@ -41,19 +37,25 @@ __Note: You can continue to push changes to your fork branch after you open a pu
 
 Once your pull request is approved, usually by at least two other community members, it will be merged into the fortran-lang.github.io master branch by the maintainers at which point it will be published to the fortran-lang.org site.
 
+If required, the repository maintainers can build a public preview of your proposed changes which will be available to view at `fortran-lang.org/pr/<pr_id>/` where `<pr_id>` is the numeric identifier of your pull request.
 
-### Pull request previews
+This allows reviewers to directly view the generated result of your PR.
+After a pull request has been merged and successfully rendered, the maintainers will delete the preview build.
 
-Once you open a pull request, a github action will execute and build your pull request branch to produce a public preview which will be available to view at `fortran-lang.org/pr/<pr_id>/` where `<pr_id>` is the numeric identifier of your pull request.
 
-This allows reviewers to directly view the generated result of your PR.
+## Style guide
 
-__Note: Subsequent pushes to your pull request branch will trigger new builds of the pull request preview.__
+### External links
 
-__Note: to disable pull request preview builds, place the string '#no_preview' within the pull request description.__
+It is recommended practice for off-site hyperlinks to open in a new tab.
+On `Fortran-lang.org` all such links will automatically be suffixed with a new-tab icon;
+this gives site users prior expectation that the link will lead them off-site while
+keeping fortran-lang.org open in a previous tab.
 
-After a pull request has been merged and successfully rendered, the preview build can be deleted by commenting on
-the pull request with the following keyword: `#delete_preview`.
+__Example:__ Open link in new tab (HTML or markdown)
+```html
+<a href="https://fortran-lang.discourse.group/" target="_blank">Discourse</a>
```

Comment:
```suggestion
<a href="https://fortran-lang.discourse.group/" target="_blank" rel="noopener">Discourse</a>
```

There's potential security issues without the `rel="noopener"` because the newly opened page has access to the originating page.

Reply:
Wow, I did not know about that! Thanks :+1: